src/app/src/components/Layers/ReferenceImageLayer.js

```diff
@@ -20,21 +19,32 @@ const convertCornerFromStateFormat = corner => ({
     lng: corner[1],
 });
 
-export default function ReferenceImageLayerVisbilityWrapper() {
-    const showLayer = useSelector(state => state.map.referenceImage.visible);
-
-    return showLayer ? <ReferenceImageLayer /> : null;
-}
-
-function ReferenceImageLayer() {
+export default function ReferenceImageLayer() {
     const dispatch = useDispatch();
+    //    const map = useRef(L.distortableCollection({ editable: true }));
```

Comment:
Consider deleting, or adding an explanation of why we're keeping it commented out.

Reply:
This was from a previous iteration and isn't needed here. Deleted in b99da628c54e72fd5a21f082b5aada77a15dee5c.